tracing/dubbo/go-client/cmd/client.go

```diff
@@ -50,19 +51,25 @@ import (
 )
 
 var (
+	userProvider        = new(pkg.UserProvider)
```

Comment:
In go, it is not a good program habit to use the keyword `new`. Use the `&pkg.UserProvider{}` instead pls.